pom.xml

```diff
@@ -24,6 +24,8 @@
   </licenses>
 
   <properties>
+    <revision>1.9.0-beta-1</revision>
```

Comment:
@batmat with this change, no need run `mvn versions:set -DnewVersion=1.9.0-SNAPSHOT`? This we do to switch from beta to final release. So the only change we need to make it edit pom to change revision property, right?

Reply:
Good point. I've never tried it on a multimodule project, but given my Maven foo: yes I do not see anything that would be needed. That's another good point of the [Maven CI Friendly versions](https://maven.apache.org/maven-ci-friendly.html) I had overlooked: no need to update all children since they now use a variable that does not need to be changed when parent is bumped, given standard inheritance (adapted in 3.5+ to support variables) will apply.

WDYT @olamy ?

Reply:
correct just edit this one. I don't remember how this will work with release plugin..
but just edit commit and tag and ` mvn deploy ` then change revision to next snapshot so only release process change a bit

Reply:
See [guide](https://github.com/jenkinsci/incrementals-tools/blob/master/README.md#enabling-production-of-incrementals). I do not see any issues here.

Reply:
@jglick -- just to be clear, `mvn release` works normally with these changes?

Reply:
@kzantow yes, you can still use the normal release process using the release plugin. If the [parent pom is ≥3.18](https://github.com/jenkinsci/plugin-pom/blob/master/CHANGELOG.md#318) it will automatically be "reincrementalified" when using common m-release-p releases. 

So, as [Blue Ocean is already on 3.23](https://github.com/jenkinsci/blueocean-plugin/blob/a6b4aa390417e7307dfb98bfa29b6d2094a0674b/pom.xml#L8), you're good to go.